src/electionguard_tools/scripts/sample_generator.py

```diff
@@ -78,18 +78,18 @@ def generate(
         # Configure the election
         # TODO: pass the spec version and the manifest name in
         (
-            manifest,
+            public_data,
```

Comment:
This was incorrectly named and initially was confusing leading to `manifest.manifest` in this PR.